docker-compose/docker-compose.base.yml

```diff
@@ -120,11 +69,14 @@ services:
     ports:
       - 8080:80
     volumes:
-      - $PWD/../../k8s/base/config/network-store-server-application.yml:/config/specific/application.yml:Z
-      - $PWD/../../k8s/base/config/common-application.yml:/config/common/application.yml:Z
-      - $PWD/../cassandra.properties:/config/cassandra.properties:Z
+      - ../k8s/base/config/network-store-server-application.yml:/config/specific/application.yml:Z
+      - ../k8s/base/config/common-application.yml:/config/common/application.yml:Z
+      - ./cassandra.properties:/config/cassandra.properties:Z
+    restart:
```

Comment:
Cassandra takes 1 minute to start and all services using cassandra fail to start
It's to avoid to start manually Cassandra before the others services